### ISSUE TITLE
No way to format heading or spacing of individual charts within Waffle

### ISSUE THREAD

**Issue by Tim-Bader** [kiewic/PowerBI-WaffleChart#18]
Feature request:
I like the idea of the Waffle chart and end-users really like the idea too, but I cannot use it in its current form, because there is no way to alter or format the headings of the individual charts or alter the spacing between them.

For my current use case, the headings for the applicable categories are quite long, so they overlap each other, which looks horrible and is unreadable.

Would you please add formatting options, so we can update the text size, and alter the spacing between charts, within the Waffle chart visual?
TIA!

**Comment by TLHarding** [comment]
I am having the same issue. I like the visual but the text overlapping is a deal-breaker for the end-user.

**Comment by Tim-Bader** [comment]
> I am having the same issue. I like the visual but the text overlapping is a deal-breaker for the end-user.

Yes, I got around it on the chart I was using by adding an acronym column to the dimension table and adding that to the chart instead, but of course, that's not always possible or appropriate.
